docs/getting_started.ipynb

```diff
@@ -0,0 +1,623 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f69e7d01",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-07-04T15:29:32.216769Z",
+     "start_time": "2022-07-04T15:29:32.177170Z"
+    },
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "from mapomatic import deflate_circuit, evaluate_layouts, matching_layouts\n",
+    "\n",
+    "trot_circ_def = deflate_circuit(trot_circ_t)\n",
+    "layouts = matching_layouts(trot_circ_def, backend)\n",
+    "scored_layouts = evaluate_layouts(trot_circ_def, layouts, backend) # cost_function = cost_func\n",
+    "print(scored_layouts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "abf874a1",
+   "metadata": {},
+   "source": [
+    "## Pulse Scaling\n",
+    "\n",
+    "For certain problems, in particular Trotterized quantum simulation problems, or other algorithms that require small angles of rotation in the two-qubit Hilbert space, it is more efficient to implement operations in terms of pulses extracted from the CNOT gate. Basically, a CNOT gate is *locally-equivalent* to an $R_{ZX}(\\pi/2)$ rotation, meaning it is built from that and single-qubit rotations, as can be seen from the following code:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d07c1600",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-07-04T15:29:32.543299Z",
+     "start_time": "2022-07-04T15:29:32.219802Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# TODO: check out Weyl decomp stuff\n",
+    "\n",
+    "qc = QuantumCircuit(2)\n",
+    "qc.cx(0, 1)\n",
+    "qc_rzx = transpile(qc, basis_gates=[\"sx\", \"rz\", \"rzx\"])\n",
+    "\n",
+    "fig, (ax1, ax2) = plt.subplots(1, 2)\n",
+    "qc.draw(\"mpl\", ax=ax1)\n",
+    "qc_rzx.draw(\"mpl\", ax=ax2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d81f1805",
+   "metadata": {},
+   "source": [
+    "The `RZXGate` is very similar to the native two-qubit interation called [echoed cross resonance](https://arxiv.org/abs/1603.04821) which is used to create entanglement on IBM backends. In particular, many two-qubit interactions for quantum simulation, such as the $ZZ$-interaction of our Ising Hamiltonian, can be more efficiently represented (in terms of error) by $R_{ZX}(\\theta)$ rotations, which are automatically broken into scaled echoed cross resonance `secr` gates unless `unroll_rzx_to_ecr` is set to `False`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b6365ce4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_research.utils.convenience import scale_cr_pulses\n",
```

Comment:
Please note in the main text that the library includes transpilation passes which the user can use to construct their own pass managers instead of using the convenience functions.